EventKit/eventkit-cloud
# Update Babel

## Description of Improvement

Update babel and related packages, update babel config.  Removes some now unnecessary packages.

## How to test

Pull down the changes and bring down your webpack container. Rebuild webpack using docker-compose build --no-cache webpack. Bring your webpack container back online. Ensure that it builds properly and that you can access the application without any issues. 

## Quality Assurance 

The following items have been updated:
 
## Screenshots

:smile:
